README.md

```diff
@@ -26,6 +26,12 @@ instances running inside your VMs.
 For both the modes to work, a user has to install and start Open vSwitch in
 each VM/host that he plans to run his containers.
 
+Installing Open vSwitch is out of scope of this documentation.
+You can read [INSTALL.Debian.md] or [INSTALL.RHEL.md] or [INSTALL.Fedora.md]
+for that and then install OVN related packages along with OVS packages.
```

Comment:
You could also just link to INSTALL.md, which then links to the platform specific guides at the top.